.travis.yml

```diff
@@ -48,8 +48,8 @@ before_install:
 
 install:
 # Installl requirements
-  - pip install xcffib
```

Comment:
We should keep this here to be sure xcffib is always installed before cairocffi.